components/brave_wallet/browser/wallet_data_files_installer.cc

```diff
@@ -227,6 +260,20 @@ void HandleParseOnRampCurrenciesLists(base::FilePath absolute_install_dir,
       base::BindOnce(&OnSanitizedOnRampCurrenciesLists));
 }
 
+void ParseCoingeckoIdsMapAndUpdateRegistry(const base::FilePath& install_dir) {
+  // On some platforms (e.g. Mac) we use symlinks for paths. Convert paths to
+  // absolute paths to avoid unexpected failure. base::MakeAbsoluteFilePath()
+  // requires IO so it can only be done in this function.
+  const base::FilePath absolute_install_dir =
+      base::MakeAbsoluteFilePath(install_dir);
+
+  if (absolute_install_dir.empty()) {
+    LOG(ERROR) << "Failed to get absolute install path.";
+  }
```

Comment:
should return in case of error?

Reply:
https://github.com/brave/brave-core/pull/19489/commits/1d5abf15734bf008bade168968c903714b7e91f7